poem/src/web/compress.rs

```diff
@@ -71,6 +75,12 @@ impl CompressionAlgo {
                     level.unwrap_or(CompressionLevel::Default),
                 ),
             ),
+            CompressionAlgo::ZSTD => Box::pin(
+                async_compression::tokio::bufread::ZstdEncoder::with_quality(
+                    BufReader::new(reader),
+                    level.unwrap_or(CompressionLevel::Fastest),
```

Comment:
copied `Fastest` from brotli - the two others are `Default`. Not sure which makes the most sense as fallback?